### PR TITLE
sg migration: Remove schemadoc

### PR DESCRIPTION
**Pull request by efritz**
Replace schemadoc with `sg migration ...` commands.

## Test plan

CI checks go generate/git diff.

### CODE REVIEW

File: internal/database/gen.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+./tmp-sg migration squash-all -skip-teardown -db frontend -f migrations/frontend/squashed.sql
+./tmp-sg migration squash-all -skip-teardown -db codeintel -f migrations/codeintel/squashed.sql
+./tmp-sg migration squash-all -skip-teardown -db codeinsights -f migrations/codeinsights/squashed.sql
+
+export PGDATASOURCE="postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}:${PGPORT}/sg-squasher-frontend"
+export CODEINTEL_PGDATASOURCE="postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}:${PGPORT}/sg-squasher-codeintel"
+export CODEINSIGHTS_PGDATASOURCE="postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}:${PGPORT}/sg-squasher-codeinsights"
+
+# Output-psql formatted schema description
+./tmp-sg migration describe -db frontend --format=psql -force -out internal/database/schema.md
```
Comment (camdencheek) [inline internal/database/gen.sh:27]: Wow, am I reading this right that the `sg migration describe` didn't change the output `schema.md` at all?

**Review by camdencheek — APPROVED** [review]

File: internal/database/gen.sh (hunk 1 above)
Comment (efritz) [reply internal/database/gen.sh:27]: Yeah, I re-implemented the psql output a while ago: #33006

Comment (camdencheek) [reply internal/database/gen.sh:27]: Nice!

**Comment by keegancsmith** [comment]
Two benefits of schema doc in repo:
1. you can grep for database schema stuff (I literally used this today)
2. you can code review the effect of migrations

Not a blocker that we are losing it, just calling out maybe unexpected benefits of it which didn't go into the decision of removing it. Is there anywhere in our codebase now where we get those side effects?

**Comment by efritz** [comment]
@keegancsmith We just removed the shell of the command that has moved to `sg migration ...`. We still generate for each schema:

- fully squashed migration SQL file
- a json-encoded version of the schema
- a psql-encoded dump of the schema

No repo content was removed here.

**Comment by keegancsmith** [comment]
great!

**Pull request merged**